docs/changelog/1_0_a3.rst

```diff
@@ -0,0 +1,158 @@
+===========
+1.0 Alpha 3
+===========
+
+This changelog summarizes new features and breaking changes in
+EdgeDB 1.0 alpha 3.
+
+
+Rust
+====
+
+We've started the process of rewriting some of the
+performance-critical EdgeDB internals to Rust. So far we have
+converted the lexer as well as the GraphQL parser.
+
+Additionally, we're also making EdgeDB command-line tools using Rust.
+Notably the REPL has been converted and now features more commands:
+
+.. code-block::
```

Comment:
I don't think we need to include this print-out.  Besides, there are only a few new commands (dump/restore?)

Reply:
History, edit, set, recent error are also new. Plus the commands themselves changed (and we've had them listed last time).

Reply:
I'd just enumerate the new commands then, briefly. Like "there are new commands to manage and navigate history: `\blah1`, `\blah2`"